### PR TITLE
Add YAML for Radarsat-1 to Registry

### PR DESCRIPTION
**Pull request by keballantyne**
Added YAML file for Radarsat-1 imager for the AWS Registry.

*Issue #, if available:* n/a

*Description of changes:*

Submission of YAML file for Radarsat-1 imagery to the AWS Registry.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: e70fe27e38dc4bbdf49eea0330bf419fad5ef4a1
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=780491ee-a546-4e7a-8a39-97780f2823b6%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/radarsat-1.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+Name: RADARSAT-1 OPEN DATA COG PRODUCTS
+Description: Developed and operated by the Canadian Space Agency, it is Canada's first commercial Earth observation satellite.
+Documentation: https://mdacorporation.com/geospatial/international/satellites/RADARSAT-1/
+Contact: https://www.eodms-sgdot.nrcan-rncan.gc.ca
+ManagedBy: "[Natural Resources Canada](https://nrcan.gc.ca/)"
+UpdateFrequency: The initial product release includes the 36000 products originally ordered by the Government of Canada. Products are added on an adhoc basis driven by prioritized foreign repatriation efforts and new processing orders from the raw archive.
+Tags:
+  - earth observation
+  - global
+  - government spending
```
Comment (jflasher) [inline datasets/radarsat-1.yaml:10]: `government-spending` tag doesn't seem appropriate? Also, please add `aws-pds` as a tag.

File: datasets/radarsat-1.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+Name: RADARSAT-1 OPEN DATA COG PRODUCTS
```
Comment (jflasher) [inline datasets/radarsat-1.yaml:1]: I think you can just change to "RADARSAT-1" and that should be sufficient here.

File: datasets/radarsat-1.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  - earth observation
+  - global
+  - government spending
+  - ice
+  - agriculture
+  - sustainability
+  - disaster response
+  - satellite imagery
+  - geospatial
+License: [Open Government License (OGL)](https://open.canada.ca/en/open-government-licence-canada)
```
Comment (jflasher) [inline datasets/radarsat-1.yaml:17]: Please wrap the markdown in "".

File: datasets/radarsat-1.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      URL: https://www.radarstac.com
+      AuthorName: Sparkgeo Consulting Inc. 
+      AuthorURL: https://sparkgeo.com/
+    - Title: QGIS
+      URL: https://qgis.org
+      AuthorName: German QGIS user group
+      AuthorURL: https://www.qgis.de/
+    - Title: GDAL
+      URL: https://gdal.org
+      AuthorName:
```
Comment (jflasher) [inline datasets/radarsat-1.yaml:42]: For each of these, you will need at least an AuthorName (can be a company name).

**Review by jflasher — CHANGES_REQUESTED** [review]
Hi @keballantyne, thanks for the PR! I've added a few comments that'll need to be addressed, but once they're fixed, I think this should be good to go.

**Pull request closed without merging**